src/parquet/column/reader.h

```diff
@@ -102,7 +102,7 @@ class PARQUET_EXPORT ColumnReader {
 
 // API to read values from a single column. This is the main client facing API.
 template <typename DType>
-class PARQUET_EXPORT TypedColumnReader : public ColumnReader {
+class TypedColumnReader : public ColumnReader {
```

Comment:
Thank you for catching this. Having `PARQUET_EXPORT` in this place triggers msvc compilation error that extern template class is not compatible with __dllexport. I'm going to solve this by structure the code in similar way how it's done in Arrow: separate header file type_fwd.h will contain forward definition of `TypedColumnReader` without `PARQUET_EXPORT`, and this forward declaration will be resolved by compiler on processing of extern template class definition. So, no warning generated.